File: week7/method.java
public class method{

    public void value(int s){
        System.out.println("the int data type: "+s);
    }
    public void value(String s){
        System.out.println("the string data type: "+s);
    }
    public void value(double s){
        System.out.println("the double data type: "+s);
    }
    public static void main(String[] args) {
        method obj=new method();
        obj.value(12);
        obj.value("sravani");
        obj.value(444);
    }
}

File: week7/method.cpp
#include<iostream>
using namespace std;
class methodoverloading{
    public:
    void value(int s){
        cout<<"the int data type: "<<s<<endl;
    }
    void value(string s){
        cout<<"the string data type: "<<s<<endl;
    }
    void value(double s){
        cout<<"the double data type: "<<s<<endl;
    }
};
int main()
{
    methodoverloading obj;
    cout<<"enter an integer: ";
    int a;
    cin>>a;
    obj.value(a);
    obj.value("sravani");
    obj.value(2.22);
}
